src/main/java/org/sagebionetworks/web/client/widget/footer/FooterViewImpl.java

```diff
@@ -68,6 +70,12 @@ public interface Binder extends UiBinder<Widget, FooterViewImpl> {}
   @UiField
   Image alphaModeStatusIcon;
 
+  @UiField
+  FocusPanel sageLogo1;
+
+  @UiField
+  FocusPanel sageLogo2;
```

Comment:
Do you mind renaming these so it's more clear that they are conditionally shown based on breakpoints?